menu.js

```diff
@@ -0,0 +1,13 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navbar_menu');
+const navBar = document.querySelector('.navbar');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  navBar.classList.toggle('active');
+});
+document.querySelectorAll('.navbar_menu_link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
- [ ] Great job working on the menu functionality so far but I think it would be great if you make the height the same as the given figma since in Figma it has a height in the whole screen but I just can see the menu above so please fix the styles to be the same as the given design ✨🔥✅



- [ ]  It should cover the secreen with a white background and please don't forget to remove the scrolling since the user should not scroll while the menu is open 🔥🙏🏻🚀

- Your

![image](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/58553711/c3ed1da8-6e4c-4ded-974d-6f0670e038c1)


- Given
![image](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/58553711/98eee0c3-8a48-474a-8a65-a5291446eca6)

Reply:
hey,I've tried overflow:hidden on navbar menu but on my side it's still scrolling.
Please help me with other suggestions.
Thank you.